### ISSUE TITLE
分割分类

### ISSUE THREAD

**Issue by Keven-Don** [ultralytics/ultralytics#19808]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

a.May i increase the cls  weight in the yolov8 instance seg task , b. The right and left  page of the book does not perform well, how can I optimize

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Keven-Don, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can explore various [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, including guidance on segmentation tasks. Many common questions may already have answers there.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate the issue.

If this is a custom training ❓ Question, please share more details about your use case, including dataset examples, training configurations, and logs. Additionally, check out our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to help optimize your model's performance.

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://reddit.com/r/Ultralytics) for more insights.

## Upgrade

Ensure you are using the latest `ultralytics` package and dependencies in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). This can help verify if your issue is resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

You can run YOLO in any of the following verified environments with necessary dependencies installed:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. These tests validate the proper functioning of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response to guide you in the right direction. An Ultralytics engineer will review your issue and assist you further shortly. Thank you for your patience! 😊

**Comment by glenn-jocher** [comment]
@Keven-Don for part (a), you can adjust classification weights by modifying the `cls_loss` gain in your hyperparameters file or via arguments during training. See the YOLOv8 Training Settings for details. 

For part (b), consider:  
1. Adding more diverse page angle examples in your training data  
2. Applying perspective/augmentation transforms  
3. Using larger models like YOLOv8m-seg  
4. Increasing input resolution  

We recommend reviewing the Instance Segmentation Guide for optimization strategies. For direct implementation help, please share a minimum reproducible example of your issue.
